configs/CMakeLists.txt

```diff
@@ -55,8 +55,23 @@ if( VIAME_DOWNLOAD_MODELS )
         ${VIAME_DOWNLOAD_DIR}/models-mmdet_generic-v1.4.tar.gz
         ${CMAKE_CURRENT_SOURCE_DIR}/pipelines )
     endif()
+    elseif( VIAME_ENABLE_PYTORCH-CUTLER )
+      DownloadAndExtract(
+        https://viame.kitware.com/api/v1/item/646b5af78790390715ac66a7/download
+        ${VIAME_DOWNLOAD_DIR}/VIAME_PoolCar_10_shot_models-V1.0.zip
+        ${CMAKE_CURRENT_SOURCE_DIR}/pipelines )
+    endiff()
```

Comment:
This if statement is malformed, elseif outside of if else block and endiff instead of endif, I fixed it on my push